changes/add_qc_failed_field.md

```diff
@@ -0,0 +1 @@
+Field qcFailed on shesmu detailed cases
```

Comment:
The field is on sequencing items within Shesmu detailed cases - not on the cases themselves